sdk/purview/azure-analytics-purview-workflow/pom.xml

```diff
@@ -0,0 +1,91 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
Add copyright text to the pom file. See https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/azure-data-appconfiguration/pom.xml#L1

Reply:
Fixed, thanks

Reply:
Copyright text is still not included in the pom file.